src/components/TutorialCarouselDialog/index.js

```diff
@@ -60,11 +60,40 @@ export default class TutorialCarouselDialog extends Component {
       <Dialog open={openTutorial} fullWidth maxWidth="md">
         <DialogContent className={cx(classes[`tutorialDialog${currentIndex}`], classes.tutorialDialog)}>
           <div className={classes.titleTopLine}></div>
-          <CurrentComponentName />
-          <div className={classes.buttonsWrapper}>
-            {currentIndex > 0 && <Button className={classes.button} variant="raised" size="medium" onClick={this.prevSlide}>Previous</Button>}
-            {currentIndex < this.components.length - 1 && <Button className={classes.button} variant="raised" size="medium" onClick={this.nextSlide}>Next</Button>}
-            {currentIndex === this.components.length - 1 && <Button className={classes.button} variant="raised" size="medium" onClick={this.closeTutorial}>Got It. Let&apos;s Start.</Button>}
+          <div className={classes.contentWrapper}>
+            <CurrentComponentName />
+            <div className={classes.buttonsWrapper}>
+              {currentIndex > 0 &&
+                <Button
```

Comment:
do you think we could do the following
```js
import { Button as _Button, withStyles } from 'material-ui';

const Button = withStyles(styles)(({ classes, ...props }) => (
  <_Button 
    {...props}
    className={classes.button} 
    variant="raised"
    size="medium"
  />
));

<Button onClick={this.prevSlide}>Previous</Button>
<Button onClick={this.nextSlide}>Next</Button>
<Button onClick={this.closeTutorial}>Got It. Let&apos;s Start.</Button>
```

Reply:
good refactor. done. also added translations.